testing_and_setup/compass/ocean/cosine_bell/RMSE/default/compute_rmse.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+import xarray as xr
+import matplotlib
+import matplotlib.pyplot as plt
+import os 
+
+%matplotlib inline
```

Comment:
I haven't seen this one before.

Reply:
this was an error in importing from jupyter notebook to python script.  It's ipython magic.  I've removed this.